keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -651,12 +668,17 @@ _This section must be completed when targeting beta graduation to a release._
   checking if there are objects with field X set) may be a last resort. Avoid
   logs or events for this purpose.
 
+The source of `/metrics/cadvisor` is the CRI implementation, not cAdvisor. Further, if the CRI implementation was using the
```

Comment:
The source of the "new" prometheus endpoint will be CRI ...

Reply:
I reworeded a bit